java/ql/src/semmle/code/java/dataflow/internal/rangeanalysis/SignAnalysisSpecific.qll

```diff
@@ -53,6 +53,12 @@ private module Impl {
   private import SignAnalysisCommon
   private import SsaReadPositionCommon
 
+  class UnsignedNumericType extends NumericOrCharType {
```

Comment:
This class is already defined in `Type.qll` with the name `CharacterType`, so `UnsignedNumericType` could just alias that.